components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/operator-finder/operator-finder.js

```diff
@@ -343,9 +388,31 @@ define(['jquery', 'lodash', 'log','remarkable', 'handlebar', 'designViewUtils','
             }
         };
 
+        /**
+         * check the whether operator extension is partially installed or not.
+         * @param operatorExtension
+         * @returns {boolean}
+         */
+        var isPartialInstalledExtension = function (operatorExtension) {
+            var partiallyInstalledExtensionArray = getPartiallyInstalledExtensionDetails();
+            for (var extension of partiallyInstalledExtensionArray) {
+                if (((operatorExtension.name.trim().toLowerCase()).indexOf(extension.extensionInfo.name.trim().toLowerCase())) > -1) return true;
```

Comment:
do not use if clauses without pranthesis even though u can write it in one line. its hard to read the code